src/backend/dx12/src/lib.rs

```diff
@@ -1036,16 +1036,20 @@ impl hal::Instance for Instance {
                     Features::INSTANCE_RATE |
                     Features::SAMPLER_MIP_LOD_BIAS,
                 limits: Limits { // TODO
-                    max_texture_size: 0,
+                    max_image_1d_size: d3d12::D3D12_RESOURCE_DIMENSION_TEXTURE1D as _,
```

Comment:
these enums doesn't makes sense in this context.
these sgould be used instead similar to the dx11 implementation: https://retep998.github.io/doc/winapi/d3d12/constant.D3D12_REQ_TEXTURE1D_U_DIMENSION.html